test/validation/api/buffer/buffer.c

```diff
@@ -537,7 +537,7 @@ static void buffer_test_user_area(void)
 
 	for (i = 0; i < num; i++) {
 		odp_event_t ev;
-		int flag;
+		int flag = 0;
```

Comment:
According to the API spec, the user flag value should be always written by `odp_event_user_area_and_flag()`. Is this actually an implementation issue or doesn't some compiler recognize the initialization properly?

Reply:
Its a compiler issue. It comes up when compiling with specific flags.

--enable-shared --enable-static --enable-lto --enable-test-vald 'CFLAGS=-O3 -ggdb -fno-var-tracking-assignments' 'CXXFLAGS=-O3 -ggdb -fno-var-tracking-assignments'

Reply:
Okay, commit message could then state that the compiler errors were invalid in this case.